Makefile

```diff
@@ -150,6 +150,12 @@ synclet-release:
 	docker push $(SYNCLET_IMAGE):$(TAG)
 	sed -i 's/var SyncletTag = ".*"/var SyncletTag = "$(TAG)"/' internal/synclet/sidecar/sidecar.go
 
+custom-synclet-release:
+	$(eval TAG := $(if $(SYNCLET_TAG),$(SYNCLET_TAG),$(shell date +v%Y%m%d)))
+	docker build -t $(SYNCLET_IMAGE):$(TAG) -f synclet/Dockerfile .
+	docker push $(SYNCLET_IMAGE):$(TAG)
+	sed -i 's/var SyncletTag = ".*"/var SyncletTag = "$(TAG)"/' internal/synclet/sidecar/sidecar.go
```

Comment:
hmmm...this `sed` rule doesn't make sense in your use-case, right? like you're not releasing your own tilt binaries with this custom image.

mostly trying to make sure i'm not totally misunderstanding your use-case

Reply:
I saw that, but if it's the same tag, which would normally be the case, the sed command is a no-op. I can take it out; I'm not sure what anyone else with a custom registry would want to do. :-)

Reply:
Removed